src/java/org/apache/cassandra/repair/messages/PrepareMessage.java

```diff
@@ -106,6 +116,7 @@ public void serialize(PrepareMessage message, DataOutputPlus out, int version) t
             out.writeLong(message.timestamp);
             out.writeBoolean(message.isGlobal);
             out.writeInt(message.previewKind.getSerializationVal());
+            out.writeInt(message.parallelism.getSerializationVal());
```

Comment:
I know we say repair isn't supported during a rolling upgrade; but this will actually break that =(